web/modules/custom/dpl_recommender/src/Controller/DplRecommenderController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Drupal\dpl_recommender\Controller;
+
+use Drupal\Core\Block\BlockManagerInterface;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Render\RendererInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+/**
+ * Render recommender react app.
+ */
+class DplRecommenderController extends ControllerBase {
+
+  /**
+   * DplRecommenderController constructor.
+   *
+   * @param \Drupal\Core\Block\BlockManagerInterface $blockManager
+   *   Drupal block manager.
+   * @param \Drupal\Core\Render\RendererInterface $renderer
+   *   Drupal renderer service.
+   */
+  public function __construct(
+    private BlockManagerInterface $blockManager,
+    private RendererInterface $renderer
+  ) {
+  }
```

Comment:
Is this constructor deliberately empty. If yes please comment why.

Reply:
No comments need this is the std. constructor property promotion pattern